typings/index.d.ts

```diff
@@ -277,6 +277,7 @@ export class Command {
   processedArgs: any[];
   commands: Command[];
   parent: Command | null;
+  readonly options: ReadonlyArray<Readonly<Option>>;
```

Comment:
The `readonly` is a reasonable suggestion, but the `options` and `commands` properties are very similar. I would prefer to keep them symmetrical, and not make the options readonly.